src/proc_image_processing/filters/white_filter.h

```diff
@@ -44,7 +44,6 @@ namespace proc_image_processing {
     private:
         cv::Mat output_image_;
 
-        Parameter<bool> enable_;
```

Comment:
The variable "enable_" was not used in this filter. We should be vigilant about its behaviour when adding the condition by calling Execute() on it. Try to verify if it works correctly with this change.